src/api/protectedApiClient.ts

```diff
@@ -11,6 +12,15 @@ export interface ProtectedApiClient {
     currentPassword: string;
     newPassword: string;
   }) => Promise<void>;
+
+  readonly registerTickets: (request: {
+    lineItemRequests: [
+      {
+        eventId: number;
+        quantity: number;
+      },
+    ];
+  }) => Promise<AxiosResponse<any>>;
```

Comment:
Is there an interface for this request anywhere? if so we should reference it here. If not we should define one.